content/en/account_management/saml/activedirectory.md

```diff
@@ -11,104 +11,25 @@ further_reading:
   tag: "Documentation"
   text: "Configuring Teams & Organizations with Multiple Accounts"
 ---
-The Datadog SAML integration for SSO provides an easy pathway for linking an organization to an external user management system so that credentials can be kept and managed in a central system.
 
-This article is meant to be used as an add-on to the main guide on this integration which is available by clicking the link below and focuses on some extra steps that may be required when connecting Datadog to ADFS.
+The Datadog SAML integration for SSO provides an easy pathway for linking an organization to an external user management system so that credentials can be kept and managed in a central system. This doc is meant to be used as an add-on to [Single Sign On With SAML (main doc)][1] doc, which gives an overview of single sign-on from the Datadog perspective.
```

Comment:
```suggestion
The Datadog SAML integration for SSO provides an easy pathway for linking an organization to an external user management system so that credentials can be kept and managed in a central system. This doc is meant to be used as an add-on to the main [Single Sign On With SAML][1] documentation, which gives an overview of single sign-on from the Datadog perspective.
```